src/tests/ui/planner/workitem-quickadd.ts

```diff
@@ -43,4 +43,13 @@ export class WorkItemQuickAdd extends ui.BaseElement {
       return filtered;
     }, []);
   }
+  
+  async addAndOpenWorkItem({ title, description = '', type = 'feature' }: WorkItem) {
+    await this.clickWhenReady();
+    await this.titleTextInput.ready();
+    await this.titleTextInput.enterText(title);
+    await this.addButton.untilClickable();
```

Comment:
We can skip this, can't we?
We want to click on the `clickAndOpen` button so waiting for this button could be skipped.

Reply:
done